File: app/src/main/java/me/dara/memoapp/ui/view/ProgressDialog.java
package me.dara.memoapp.ui.view;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatDialogFragment;
import me.dara.memoapp.R;

/**
 * @author sardor
 */

/**
 * Custom progress
 */
public class ProgressDialog extends AppCompatDialogFragment {

  @Nullable @Override
  public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
      @Nullable Bundle savedInstanceState) {
    return inflater.inflate(R.layout.dialog_loading, container, false);
  }
}

File: app/src/main/java/me/dara/memoapp/ui/view/ProgressDialog.kt
package me.dara.memoapp.ui.view

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.appcompat.app.AppCompatDialogFragment
import me.dara.memoapp.R

/**
 * @author sardor
 */
class ProgressDialog : AppCompatDialogFragment() {

  override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
                            savedInstanceState: Bundle?): View? {
    return inflater.inflate(R.layout.dialog_loading, container, false)
  }




}